tests/test_density_matrix.py

```diff
@@ -156,3 +158,20 @@ def test_remove_low_amplitude():
     dm.remove_low_amplitude(1e-1)
     assert dm.mat.nnz == 1
     assert dm.mat.trace() == pytest.approx(1)
+
+
+def test_photon_loss():
+
+    dm = DensityMatrix.from_svd(BasicState([5]))
+    dm.apply_loss(0, .5)
+
+    assert dm.mat.trace() == pytest.approx(1)
+
+    for k in range(6):
+        assert dm[BasicState([k]), BasicState([k])] == pytest.approx(math.comb(5, k) * (1/2)**5)
+
+    sv = BasicState([2, 1, 0]) + BasicState([1, 3, 2]) + BasicState([0, 0, 1])
+    dm = DensityMatrix.from_svd(sv)
+    dm.apply_loss([0, 1], .2)
+
+    assert dm.mat.trace() == pytest.approx(1)
```

Comment:
I agree. It's not enough of a test but I thought it was necessary at first to be sure that the computation was giving back something normal. The tests in the loop above are more precise because they check the number of loosed photon is actually following a binomial probability law (which is what is expected)

Reply:
could a test like that be added for the second case too?